src/widgets/Tabs/PatientEdit.js

```diff
@@ -55,17 +64,21 @@ const PatientEditComponent = ({
   canEditPatient,
 }) => {
   const { pageTopViewContainer } = globalStyles;
+  const [isDeceasedModalOpen, toggleIsDeceasedAlert] = useToggle(false);
   const formRef = useRef(null);
-
   const savePatient = useCallback(
     e => {
       updatePatientDetails(completedForm);
-      formRef?.current?.submit(e);
-
-      if (canSaveForm) {
-        onCompleted();
+      if (!completedForm.isDeceased) {
```

Comment:
:nit there are certainly exceptions to this.. but generally I think it reads better having positive assertions in conditionals.. in this case `if(completedForm.isDeceased) {`

:nit I prefer early returns over nested statements. in this case it could be
```
if (completedForm.isDeceased) {
  toggleIsDeceasedAlert();
  return;
}

formRef?.current?.submit(e);
if (canSaveForm) {
  onCompleted();
} else {
  ToastAndroid.show(dispensingStrings.validation_failed, ToastAndroid.LONG);
}
```
( could return early instead of the else.. but 🤷  )

:nit the `e` is unused, so we can remove it 😉

Reply:
Fixed in 7cc1233311bd64ff03002b002899677c85834c60
 >:nit the e is unused, so we can remove it 
🤔 It is used in `formRef?.current?.submit(e);`